setup.py

```diff
@@ -3,31 +3,37 @@
 import ctypes
 import sys
 import unittest
+
 try:
     from setuptools import setup, Extension, Command
 except ImportError:
     from distutils.core import setup, Extension, Command
 from distutils.command.build_ext import build_ext
 from distutils.command.build_py import build_py
-from distutils.errors import CCompilerError, DistutilsExecError, \
-                             DistutilsPlatformError
-from pike import __version__
+from distutils.errors import CCompilerError, DistutilsExecError, DistutilsPlatformError
 
 # attempt building the kerberos extension
 try_krb = True
 
-if sys.platform == 'win32' and sys.version_info > (2, 6):
-   # 2.6's distutils.msvc9compiler can raise an IOError when failing to
-   # find the compiler
-   # It can also raise ValueError http://bugs.python.org/issue7511
-   ext_errors = (CCompilerError, DistutilsExecError, DistutilsPlatformError,
-                 IOError, ValueError)
+if sys.platform == "win32" and sys.version_info > (2, 6):
```

Comment:
I think it's generally recommended to use `sys.platform.startswith(...)` instead of `sys.platform == ...`.

Reply:
i looked at it a bit close and squinted, and decided that

1) we don't support py2.6 anymore, just look at our shiny new `python_requires`
2) this thing was checking that the sys.version_info was _greater_ than `(2, 6)` and then applying the hack... so it uhh _never_ worked 😂 

-> REMOVED